client/my-sites/domains/domain-management/list/test/index.js

```diff
@@ -102,19 +104,19 @@ describe( 'index', () => {
 			}
 		);
 		const cartManagerClient = createShoppingCartManagerClient( { getCart, setCart } );
-		return mount( <SiteDomains { ...props } />, {
-			wrappingComponent: TestProvider,
-			wrappingComponentProps: { store, cartManagerClient },
+		return render( <SiteDomains { ...props } />, {
+			wrapper: ( { children } ) => (
+				<TestProvider store={ store } cartManagerClient={ cartManagerClient }>
+					{ children }
+				</TestProvider>
+			),
 		} );
 	}
 
 	describe( 'regular cases', () => {
-		beforeEach( () => {
-			component = renderWithProps();
-		} );
-
 		test( 'should list two domains', () => {
-			expect( component.find( 'DomainRow' ) ).toHaveLength( 2 );
+			renderWithProps();
```

Comment:
Nit: I wonder if we should inline the contents of this function. It's invoked just once, so it feels like it's just causing unnecessary indirection. Leaving it to your judgement, though, since it was here before this PR.